languages/csharp/exercises/concept/arrays/.docs/introduction.md

```diff
@@ -0,0 +1,42 @@
+In C#, data structures that can hold zero or more elements are known as _collections_. An **array** is a collection that has a fixed size and whose elements must all be of the same type. Elements can be assigned to an array or retrieved from it using an index, with the first element's index being zero:
+
+```csharp
+// Declare array with explicit size (size is 2)
+int[] twoInts = new int[2];
+
+// Assign first and second element by index
+twoInts[0] = 7;
+twoInts[1] = 8;
+
+// Retrieve the second element by index
+twoInts[1] == 8; // => true
+```
+
+Arrays can also be defined using a shortcut notation that allows you to both create the array and set its value. As the compiler can now tell how many elements the array will have, the length can be omitted:
+
+```csharp
+// Assign and initialize (size is 3)
+int[] threeIntsV1 = new int[] { 4, 9, 7 };
+
+// Shorter notation (size is 3)
+int[] threeIntsV2 = new[] { 4, 9, 7 };
+
+// Shortest notation (size is 3)
+int[] threeIntsV3 = { 4, 9, 7 };
+```
+
+Arrays can be manipulated by either calling an array's methods or properties, or by using the methods defined in the `Array` class.
+
+Besides access values by index, the fact that an array is also a _collection_ means that you can iterate over all its values using a `foreach` loop:
```

Comment:
```suggestion
The fact that an array is also a _collection_ means that besides accessing values by index, you can iterate over all its values using a `foreach` loop:
```

Reply:
I like the change. I'd like another comma though :)

```suggestion
The fact that an array is also a _collection_ means that, besides accessing values by index, you can iterate over all its values using a `foreach` loop:
```